src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,7 +103,8 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setMaxWidth(70);
+			getTable().getColumnExt(0).sizeWidthToFit();
```

Comment:
Not exactly, the getWidth() will not return the value that we want.

Right, that just returns the renderer, you need to obtain the renderer's component and use its max width, for example:
```Java
Component c = getTable().getTableHeader().getDefaultRenderer().getTableCellRendererComponent(null, getTable().getColumnExt(0).getHeaderValue(), false, false, 0, 0);
getTable().getColumnExt(0).setMaxWidth(c.getMaximumSize().width);
```
(although we should check and use the column's header renderer, if it has one)